genienlp/server.py

```diff
@@ -102,18 +102,36 @@ def handle_request(self, request):
         with torch.no_grad():
             if self.args.calibrator_paths is not None:
                 output = generate_with_model(self.model, [batch], self.numericalizer, task, self.args,
-                                                        output_predictions_only=True,
-                                                        confidence_estimators=self.confidence_estimators)
+                                             output_predictions_only=True,
+                                             confidence_estimators=self.confidence_estimators)
                 response = []
-                for idx, p in enumerate(output.predictions):
-                    instance = {'answer': p[0], 'score': {}}
-                    for e_idx, estimator_scores in enumerate(output.confidence_scores):
-                        instance['score'][self.estimator_filenames[e_idx]] = float(estimator_scores[idx])
-                    response.append(instance)
+                if len(self.args.num_outputs) > 1 or self.args.num_outputs[0] > 1:
```

Comment:
The correct condition would be `sum(self.args.num_outputs) > 1` since it is always an array.